src/configure/configure.ts

```diff
@@ -254,7 +264,7 @@ class Orchestrator {
     }
 
     private async analyzeNode(node: any): Promise<GenericResource> {
-        if (!!node && !!node.fullId) {
+        if (!!node && !!node.cloudResource.id) {
```

Comment:
can it break the existing right click scenarios for webApp ??

Reply:
Is the right click scenario for web app fully implemented?
I thought that is still pending, and this thing will be verified when that is being implemented.

Reply:
Agree. `cloudResource` property might not exist for web app. Better to make this check for AKS case only in extractAzureResourceFromNode